src/renderers/__tests__/ReactCompositeComponent-test.js

```diff
@@ -504,8 +504,8 @@ describe('ReactCompositeComponent', () => {
 
     expectDev(console.error.calls.count()).toBe(1);
     expectDev(console.error.calls.argsFor(0)[0]).toBe(
-      'Warning: defaultProps was defined as an instance property ' +
-        'on Component. Use a static property to define defaultProps instead.',
+      'Warning: setting defaultProps as an instance property on Component is not supported ' +
```

Comment:
Yea, I only really saw it in the code.
I thought it's worth changing as it might come up in custom warning dialog (I don't remember if it appends `Warning:` or not).